python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -132,9 +149,7 @@ class Meta:
     partitionStatusCounts = non_null_list(
         "dagster_graphql.schema.partition_sets.GraphenePartitionStatusCounts"
     )
-    assetPartitionsStatusCounts = non_null_list(
-        "dagster_graphql.schema.partition_sets.GrapheneAssetPartitionsStatusCounts"
-    )
+    assetBackfillData = graphene.Field(GrapheneAssetBackfillData)
```

Comment:
Should this be non nullable?

Reply:
I don't think it always exists though, since the asset backfill data will only exist on asset backfills, not job backfills

Reply:
Ah okay makes sense thanks! Can we also export a flag `isAssetBackfill` that will effectively check that `assetBackfillData` exists? This is so that on the backfill overview page I can link to the new asset backfill page only if it's a asset backfill.